### PR TITLE
docs: web view

### PR DESCRIPTION
**Pull request by Mangesh-Khairnar**
web-views

### CODE REVIEW

File: frappe_io/www/docs/user/en/tutorial/web-views.md [hunk 1]
```diff
@@ -1,68 +0,0 @@
-<!-- base_template: frappe_io/www/frappe/frappe_base.html --><!-- add-breadcrumbs -->
```
Comment (netchampfaris) [inline frappe_io/www/docs/user/en/tutorial/web-views.md]: If someone visits this old page, they should be redirected to the new one.

File: frappe_io/www/docs/user/en/web-views.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+<!-- add-breadcrumbs -->
+# DocType Web View
+
+Web views are customizable, dynamically generated customer facing web pages which are based on content fetched from the DocType.
+
+## Getting started
+
+- Open the DocType in editable mode
+- Add a route field and a condition field. Route field defines the document route
+- Enable web view for a DocType by setting `has web view` option in the web view section.
```
Comment (netchampfaris) [inline frappe_io/www/docs/user/en/web-views.md]:
```suggestion
- Enable web view for a DocType by setting `Has Web View` option in the web view section.
```

File: frappe_io/www/docs/user/en/web-views.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+<!-- base_template: frappe_io/www/frappe/frappe_base.html -->
+<!-- add-breadcrumbs -->
+# DocType Web View
+
+Web views are customizable, dynamically generated customer facing web pages which are based on content fetched from the DocType.
+
+## Getting started
+
+- Open the DocType in editable mode
```
Comment (netchampfaris) [inline frappe_io/www/docs/user/en/web-views.md]: End all sentences with a full stop.

File: frappe_io/www/docs/user/en/web-views.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+You can define following parameters in the list view context that can allow you to change the structure of the web page
+
+- **show_sidebar:** display sidebar
+- **show_search:** display search option
+- **no_breadcrumbs:** removes breadcrumbs
+- **title:** title of the web page
+- **list_template:** define path of a fully customized list template
+
+### Customizing Grid Element
```
Comment (netchampfaris) [inline frappe_io/www/docs/user/en/web-views.md]:
```suggestion
### Customizing Row Template
```

**Comment by ddavidebor** [comment]
Waiting on #283 before evaluating merging

**Comment by ddavidebor** [comment]
@netchampfaris can you complete the review?

File: frappe_io/www/docs/user/en/tutorial/web-views.md (hunk 1 above)
Comment (ddavidebor) [reply frappe_io/www/docs/user/en/tutorial/web-views.md:1]: @netchampfaris is there a way to do redirects?

**Pull request closed without merging**